images/linux/scripts/helpers/os.sh

```diff
@@ -6,15 +6,20 @@
 
 function isUbuntu16
 {
-    lsb_release -d|grep 'Ubuntu 16' > /dev/null
+    lsb_release -d | grep -q 'Ubuntu 16'
```

Comment:
for future cleanup maybe, should we add an "IMAGE_NAME" variable in the conf that we can use for this? seems silly that we need this.

Reply:
we should think of it

Reply:
@alepauly, With  "IMAGE_NAME" variable using multiple conditions we have got complex syntax, instead of we have now. I think we can change the approach after deprecation Ubuntu 16.04.

```
if isUbuntu16 || isUbuntu18 ; then
    add-apt-repository ppa:ansible/ansible
    apt-get update
fi
```

Reply:
Yeah, let's think of this for future cleanup, shouldn't block here.